### ISSUE TITLE
Cant start paperwork on ubuntu 14.04

### ISSUE THREAD

**Issue by peterpeter5** [jflesch/paperwork#383]
Hi,

i followed your instructions for installing paperwork on ubuntu. Unfortunately it doesnt start. Running paperwork-checkdeps results in missing dependencies. I followed the instructions from https://github.com/jflesch/paperwork/issues/300 , 
but even then the result is the same

``` paperwork-chkdeps 
[WARN] Gtk is missing !
[WARN] Glade UI is missing !
[WARN] Poppler is missing !
Traceback (most recent call last):
  File "/usr/local/bin/paperwork-chkdeps", line 345, in <module>
    missing = find_all_missing_deps()
  File "/usr/local/bin/paperwork-chkdeps", line 314, in find_all_missing_deps
    missing += check_cairo()
  File "/usr/local/bin/paperwork-chkdeps", line 258, in check_cairo
    from gi.repository import GLib
ImportError: No module named repository

**Comment by jflesch** [comment]
I've commited a fix in the branch 'stable': 1848dcc78e843f8163b898f885dcde563d944267
Can you test it please ?

**Comment by peterpeter5** [comment]
Starts and works like charm! (for what I can see from my first scans :) )
